src/UserSwitch/UserSwitch.tsx

```diff
@@ -0,0 +1,278 @@
+import React, {
+  FC,
+  MouseEvent,
+  MouseEventHandler,
+  ReactElement,
+  useState,
+} from 'react';
+import {
+  isPseudonymizedMember,
+  isSessionExpired,
+  isError,
+} from '@graasp/utils';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Note for the future: we probably want to use the prettier-sort-imports plugin with a group dedicated to material / mui 😅